lib/presentation/base/svg_provider/svg_provider.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:hive/hive.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter/material.dart';
+
+class SVGNotifier extends StateNotifier<List<Widget>> {
```

Comment:
It is not recommended to create a global state for a list of widgets. It will drastically increase memory usage. 

I recommend using List<String>. And render the widgets with this data in the screen itself. 
This way the widgets will render only when the user is viewing them.

*You can provide us some points on why if you think your implementation is the right way for the feature.

Reply:
yup! you are correct i will do that